### ISSUE TITLE
Plugin breaks when non default config/plugin folder are used

### ISSUE THREAD

**Issue by phit** [rojo8399/PlaceholderAPI#43]
SpongeAPI version: 5/7
PlaceholderAPI version: PlaceholderAPI-4.3.jar
Plugins installed: doesn't matter, but tested with VirtualChest

Error log: https://gist.github.com/phit/3183779fc09b5fef9943abcdf6e1dd06

Plugin completely breaks down when you have non default config and plugin folder paths set in your Sponge global.conf
In this case I had it set to:
```
        config-dir="${CANONICAL_GAME_DIR}/plugins/config"
        plugins-dir="${CANONICAL_GAME_DIR}/plugins"
```

Workaround is manually putting configs generated on another server in the usual `config/` folder. Then it loads fine..

**Comment by Wundero** [comment]
I will have a fix for this in the next version. I don't know what the root issue of this is that is making the config null but I have a workaround that should fix this issue for the most part.

**Comment by Wundero** [comment]
@rojo8399 Do you know why the config would be null? I can't seem to figure out what's going wrong here. Our dependencies are not attached to a file location but injected, so there shouldn't be problems like this right?

**Comment by ziceptor** [comment]
my settings

config-dir="${CANONICAL_GAME_DIR}/config"
plugins-dir="${CANONICAL_MODS_DIR}/plugins"

**Comment by Wundero** [comment]
The issue isn't tied to the directories necessarily; it seems to be an issue within the file loading that the nodes are not loaded. I am keeping this issue open because it was first but it will be fixed in 4.4.

**Comment by Wundero** [comment]
Should be fixed now, but I have no good way to test this so please check.
